sdk/tests/types/parents.rs

```diff
@@ -10,6 +11,13 @@ use iota_sdk::types::block::{
 };
 use packable::{bounded::TryIntoBoundedU8Error, error::UnpackError, prefix::VecPrefix, PackableExt};
 
+#[test]
+fn len() {
+    for i in 1..=8 {
+        assert_eq!(Parents::from_vec(rand_block_ids(i)).unwrap().len(), i);
```

Comment:
If `rand_block_ids()` ever returns the same id twice this would fail :trollface:

Reply:
You have more chances to be hit by a meteorite, twice